web/ui/navbar/index.svelte

```diff
@@ -49,28 +51,45 @@
 
 <nav class="navbar">
   <div class="navbar_main" aria-orientation="horizontal" role="menu">
-    {#if $isRefreshing}
+    <div
+      class={`navbar_back-btn ${$side === 'first' ? 'navbar_btn-hidden' : ''}`}
+    >
       <NavbarItem
-        name={$t.refresh}
-        current={$router.route === 'refresh'}
-        hotkey="r"
-        href={getURL('refresh')}
-        small
-      >
-        <NavbarProgress value={$refreshProgress} />
-      </NavbarItem>
-    {:else}
-      <NavbarItem
-        name={$t.refresh}
-        current={$router.route === 'refresh'}
-        hotkey="r"
-        icon={mdiRefresh}
+        name={$t.back}
+        current={true}
+        hotkey="Esc"
```

Comment:
Will we have 2 back-to-first step calls on small screen on pressing `Esc`?

The first one will be from `<TwoStepPage>` listener, the second will be from the button (`hotkey` is not just a visual, it creates listener).

Reply:
`Esc` is not valid name for key code, so it will no affect on `Back` button:
```js
let press = findHotKey(event, window, transformers)
if (press) press.click()
```

Reply:
In this case we should not use it, since invalid data will confuse other developers.

it is OK just remove `hotkey` here.